File: Csharp.Api/Model/ProdutoModel.cs
namespace Csharp.Api.Model
{
    public class ProdutoModel
    {
        public int IdProduto { get; set; }
        public string Nome { get; set; }
        public string Descricao { get; set; }
        public TipoProdutoModel tpProduto { get; set; }
        public int qtdProduto { get; set; }
        public float vlrProduto { get; set; }
        public DateTime dthCriadoProduto { get; set; }
        public DateTime dthAlteracaoProduto { get; set; }
    }
}

File: Angular/src/app/Models/Produto.model.ts
import { DatePipe } from "@angular/common";

export interface ProdutoModel{
    idProduto: number;
    nome: string;
    descricao:string;
    tpProduto: string;
    qtdProduto: number;
    vlrProduto: number;
    dthCriacaoProduto: Date;
    dthAlteracaoProduto: Date;
}